src/libs/Navigation/Navigation.ts

```diff
@@ -577,10 +577,7 @@ const dismissModalWithReport = ({reportID, reportActionID, referrer, backTo}: Re
             return;
         }
         dismissModal();
-        // eslint-disable-next-line @typescript-eslint/no-deprecated
-        InteractionManager.runAfterInteractions(() => {
-            navigate(reportRoute);
-        });
+        navigate(reportRoute);
```

Comment:
I think this change is too much for us at the scope of this PR 🤔 `dismissModalWithReport` is used in many places so this removal should be tested much better on slower devices too (or with CPU throotling on)

Reply:
Yes it's necessary, see https://github.com/Expensify/App/issues/74364#issuecomment-3492703213 and other comments.

TL:DR: This code was used to fix a bug that's no longer reproducible. It's necessary to remove it because that added delay is what had us add this delay in the first place (chain effect)

Reply:
It looks much better without throttling though

- with InteractionManager

https://github.com/user-attachments/assets/5c1d6ff8-2773-4640-95e3-1d438d7cbe60


- withoutInteractionManager

https://github.com/user-attachments/assets/569714f5-ea55-4658-a4e3-6d4a99ef2c5d

Reply:
@mountiny I think we need your (or maybe design team) blessing on this change

Reply:
Discussed with @war-in That this is not that simple to get around to make sure the navigation events happen in the proper order. So I tend to accept the state without waiting for the animation now given the performance gains are really noticeable and work on ways to achieve the nice ux animations in a fast follow up.

So that would mean we will have this experiencing when dismissing the modal and navigating in wide web or desktop
https://github.com/user-attachments/assets/569714f5-ea55-4658-a4e3-6d4a99ef2c5d